DesktopModules/Vanjaro/Core/Library/Factories/PageFactory.cs

```diff
@@ -18,6 +21,25 @@ public static partial class Factories
     {
         public class PageFactory
         {
+            public static ReviewContentInfo GetReviewContentInfo(int Version, string Entity, int EntityID, UserInfo UserInfo)
+            {
+                ReviewContentInfo rinfo = new ReviewContentInfo();
+                if (Entity == WorkflowLogType.VJPage.ToString())
```

Comment:
Consider changing WorkflowLogType to just Worfklow. So ultimately it would look like WorkflowType.Page